### PR TITLE
Update UG to include '0' values for height/weight

### LINKED ISSUE #287: [PE-D][Tester B] `weight`: Specifying zero weight deletes the latest weight.
Steps to recreate:

1. Person at index 1 with weight values.
2. Run `weight 1 0.0`

Result: latest weight is deleted.

Not a documented feature in the UG so I think this is a bug.

<i><video controls><source src="https://raw.githubusercontent.com/chiaryan/ped/main/files/18783f18-ceed-4c10-a912-52df4ee88dce.mp4" type="video/mp4">Your browser does not support the video tag.</video><br>video:https://raw.githubusercontent.com/chiaryan/ped/main/files/18783f18-ceed-4c10-a912-52df4ee88dce.mp4</i>

-------------
Labels: `type.FunctionalityBug` `severity.Medium`
original: chiaryan/ped#7

### PR DESCRIPTION
**Pull request by ruijietay**
Resolves #287

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -143,6 +143,8 @@ Adds a client to the FitBook.
 
 Format: `add n/NAME p/PHONE_NUMBER [e/EMAIL] [a/ADDRESS] [w/WEIGHT] [h/HEIGHT] [nt/NOTE] [t/TAG]…​`
 
+* When HEIGHT and/or WEIGHT is specified to be 0, the respective height/weight values will not be shown on the client's details.
```
Comment (jjchee77) [inline docs/UserGuide.md:146]: Would it be better to mention that it will not be added to the client?

File: docs/UserGuide.md [hunk 2]
```diff
@@ -172,6 +174,8 @@ Format: `edit INDEX [n/NAME] [p/PHONE] [e/EMAIL] [a/ADDRESS] [w/WEIGHT] [h/HEIGH
 * Existing values will be updated to the input values.
 * When no value is specified after a prefix, the value of that prefix will be removed from the client. (e.g. `edit 1 nt/` removes the note from the client at index 1).
     * Since WEIGHT can store multiple historical values, specifying an empty weight prefix removes the latest weight value from the client. Otherwise, the latest weight value will be replaced by the specified weight in this command.
+* When WEIGHT is specified to be 0, the latest weight (if any) will be removed from the client.
```
Comment (ruijietay) [inline docs/UserGuide.md:177]:
```suggestion
* When WEIGHT is specified to be 0, the **latest** weight (if any) will be removed from the client.
```

File: docs/UserGuide.md (hunk 1 above)
Comment (ruijietay) [inline docs/UserGuide.md:146]:
```suggestion
* When HEIGHT and/or WEIGHT is specified to be 0, the values will not be added to the client's details.
```

Comment (dillontkh) [inline docs/UserGuide.md:146]:
```suggestion
* When HEIGHT and/or WEIGHT is specified to be 0, no height/weight will be added to the client.
```

File: docs/UserGuide.md [hunk 3]
```diff
@@ -143,6 +143,8 @@ Adds a client to the FitBook.
 
 Format: `add n/NAME p/PHONE_NUMBER [e/EMAIL] [a/ADDRESS] [w/WEIGHT] [h/HEIGHT] [nt/NOTE] [t/TAG]…​`
 
+* When HEIGHT and/or WEIGHT is specified to be 0, the values will not be added to the client's details.
```
Comment (dillontkh) [inline docs/UserGuide.md:146]:
```suggestion
* When HEIGHT and/or WEIGHT is specified to be 0, no weight/height will be added to the client.
```

File: docs/UserGuide.md [hunk 4]
```diff
@@ -143,6 +143,8 @@ Adds a client to the FitBook.
 
 Format: `add n/NAME p/PHONE_NUMBER [e/EMAIL] [a/ADDRESS] [w/WEIGHT] [h/HEIGHT] [nt/NOTE] [t/TAG]…​`
 
+* When HEIGHT and/or WEIGHT is specified to be 0, no weight/height will be added to the client.
```
Comment (dillontkh) [inline docs/UserGuide.md:146]:
```suggestion
* When HEIGHT and/or WEIGHT is specified to be 0, no height/weight will be added to the client.
```

**Review by bgopi23 — APPROVED** [review]
LGTM!

**Review by jjchee77 — APPROVED** [review]
LGTM!

**Review by RyanNgWH — APPROVED** [review]
LGTM

**Pull request merged**